okta/resource_okta_user.go

```diff
@@ -265,6 +268,21 @@ func resourceUser() *schema.Resource {
 				Optional:    true,
 				Description: "User zipcode or postal code",
 			},
+			"password": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "User Password",
```

Comment:
I think we want this to be `Sensitive: true`. Can we also make it clear in the docs and description that this WILL be stored in plain text in state?

Reply:
It also looks like these properties are set on create only. Can we make that clear? "User's initial password. This value cannot be updated."

Technically these should be `ForceNew: true` if they can only be set on create. We should warn of that in the description.

Reply:
Technically, okta has an API that allows you to update a user's password. The go lang SDK also has an API to do this - https://github.com/okta/okta-sdk-golang/blob/master/tests/integration/user_test.go#L237

Reply:
Oh coolio, no in that case we would just need to update our update function and support that.

Reply:
Yes, you're reading it right. 😄  
I will update the `resourceUserUpdate` function as well.

Reply:
I believe I've addressed all your concerns. Take a look.